ClientApp/viking-vault/src/components/styles.css

```diff
@@ -0,0 +1,171 @@
+.sideBar-true{
+    background-color: white;
+    color: #09424d;
+    font-weight: bold;
+    font-size: 13px;
+    height: 100%;
+    transition: 0.5s;
+    position: absolute;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    width: 150px;
+}
+
+.sideBar-false{
+    background-color: white;
+    color: #09424d;
+    font-weight: bold;
+    font-size: 13px;
+    height: 100%;
+    transition: 0.5s;
+    position: absolute;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    width: 250px;
+}
+
+.topBar{
+    background-color: #09424d;
+    font-weight: bold;
+    font-size: 13px;
+    height: 60px;
+    width: 100%;
+    max-width: 1920px;
+    align-items: center;
+    vertical-align: middle;
+}
+
+.regularDiv{
+    border-bottom: 1px solid #09424d;
+    height:46px;
+}
+
+img{
+    width:20px;
+    height:20px;
+}
+
+img.vikingLogo{
+    width: 224px;
+    height: 55px;
+    position: absolute;
+    margin-left: 45%;
+    margin-right: 50%;
+}
+
+p{
+    margin-left: 5%;
+}
+
+.account-true{
```

Comment:
Maybe try to add another class name in here instead of `.something-true` or `something-false`, it should be `.something.something-else` or `.something.in-one-case` or `.something.in-another-case`